plugins/mumble_plugin_utils.h

```diff
@@ -6,6 +6,22 @@
 #ifndef MUMBLE_MUMBLE_PLUGIN_UTILS_H_
 #define MUMBLE_MUMBLE_PLUGIN_UTILS_H_
 
+#ifdef OS_LINUX
+# include <fenv.h>
+#endif
+
+#ifdef _MSC_VER
+# include <intrin.h>
+#endif
+
+// This union is used by isBigEndian() to determine the endianness.
```

Comment:
Could be turned into Doxygen docs simply by adding an additional slash to the beginning of the comment :)